test/verifiedTests/design/testSingleProductionEnvelope.m

```diff
@@ -0,0 +1,46 @@
+% The COBRAToolbox: testSingleProductionEnvelope.m
+%
+% Purpose:
+%     - test the singleProductionEnvelope function
+%
+% Authors:
+%     - Jacek Wachowiak
+global CBTDIR
+% save the current path
+currentDir = pwd;
+
+% initialize the test
+fileDir = fileparts(which('testSingleProductionEnvelope'));
+cd(fileDir);
+
+% test variables
+model = readCbModel([CBTDIR filesep 'test' filesep 'models' filesep 'ecoli_core_model.mat']);
+deletions = model.rxns(2);
+deletions2 = model.genes(2);
+product = char(model.rxns(3));
+biomassRxn = char(model.rxns(1));
+
+% function outputs
+% requires Global Optimization Toolbox
+% this function's output is a plot therefore it can be tested only by checking if the plots were created and then deleted successfully
+singleProductionEnvelope(model, deletions, product, biomassRxn, 'savePlot', 1);
```

Comment:
It might be worth modifying the function to give the resulting curve as output.
Then we could actually test it. 
The change wouldn't make any existing code fail (it will simply  not use the output) and would llow future users to get the curve values.

Reply:
@jacekwachowiak?